_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: Release 1.0.0, 1.0.1, 1.1.1
+---
+
+A new release of MDAnalysis now available!
+Or rather, three releases have been made for the 1.x version of MDAnalysis, so we just talk about all of them in a single post.
+
+
+This version brings a multitude of **fixes**, [**deprecations**](#deprecations), and **new features**.
+Some highlights are given below, whilst the release notes(someone make this a link to the right place please) list all the changes in this version.
```

Comment:
```suggestion
Some highlights are given below, whilst the [CHANGELOG](https://github.com/MDAnalysis/mdanalysis/blob/release-1.1.1/package/CHANGELOG).
```

We haven't had wiki-based release notes since 0.19.2.